src/sessionManager.ts

```diff
@@ -208,15 +208,27 @@ export class SessionManager implements ISessionManager {
 
         return this.getDeviceId()
             .then(() => {
-                let browserInfo = Utils.getBrowserInfo();
+                let platformVersion = "Unknown";
+                
+                if(typeof navigator !== "undefined"){
+                    if(navigator.product === "ReactNative"){
+                        platformVersion = "ReactNative";
+                    }else if(navigator.userAgent){
+                        let browserInfo = Utils.getBrowserInfo();
+                        if(browserInfo.version){
+                            platformVersion = browserInfo.version;
+                        }                        
+                    }    
```

Comment:
Worried the fix above will not be very future proof. Why don't we just concatenate the navigator.product and navigator.userAgent if they exist.

platformVersion = (navigator.product !== "undefined" ? navigator.product : "Unknown") + (navigator.userAgent !== "undefined" ? " : " + navigator.userAgent : "")

Reply:
done